packages/core/src/components/Modal/Modal.stories.mdx

```diff
@@ -6,9 +6,8 @@ import * as stories from './Modal.stories';
 
 # Modal
 
-Basic modal component. You can play with the component in the canvas tab.
-
-Modal consists of three sections Header, Content and Actions. All of these you can directly use from Modal component only like bellow . . .
+The `Modal` components are separate windows within an application that are mostly overlaid on the primary window. The contents behind a modal dialog are inert, meaning you cannot interact with the content behind the dialog. It is a common user interface pattern that provides information or requires confirmation.
```

Comment:
We should change it to `component is separate windows within an application that is overlaid on the primary window`